src/combinatorics/simple_graph/basic.lean

```diff
@@ -1262,6 +1262,27 @@ abbreviation comp (f' : G' ↪g G'') (f : G ↪g G') : G ↪g G'' := f.trans f'
 
 end embedding
 
+section induce_hom
+
+variables (G) (G') (G'' : simple_graph X) {s : set V} {t : set W} {r : set X}
```

Comment:
```suggestion
variables {G G'} (G'' : simple_graph X) {s : set V} {t : set W} {r : set X}
```

Reply:
I'm not sure you meant to turn `G G'` implicit, but I went with that.